sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1140,10 +1140,12 @@ public List<CaseExportDto> getExportList(
 							filteredImmunizations.sort(Comparator.comparing(i -> ImmunizationEntityHelper.getDateForComparison(i, false)));
 							Immunization mostRecentImmunization = filteredImmunizations.get(filteredImmunizations.size() - 1);
 							Integer numberOfDoses = mostRecentImmunization.getNumberOfDoses();
+							Symptoms caseSymptoms = symptoms.get(exportDto.getSymptomsId());
+							Date onsetDate = caseSymptoms != null ? caseSymptoms.getOnsetDate() : null;
 
 							List<Vaccination> relevantSortedVaccinations = vaccinationService.getRelevantSortedVaccinations(
 								filteredImmunizations.stream().flatMap(i -> i.getVaccinations().stream()).collect(Collectors.toList()),
-								exportDto.getSymptoms().getOnsetDate(),
+								onsetDate,
```

Comment:
The changes technically avoid the NPE, but `Optional.ofNullable(exportDto.getSymptoms()).map(Symptoms::getOnsetDate).orElse(null)` would do the same. But still the meaning what a relevant vaccination is changes whether you export symptoms or not. I suppose that's wrong.

Reply:
Ah I'm sorry. I rechecked your whole changes and now I see that I'm wrong. Maybe just change the assignment like this: `Optional.ofNullable(symptoms.get(exportDto.getSymptomsId())).map(Symptoms::getOnsetDate).orElse(null)`.

Reply:
done